### ISSUE TITLE
Uninstall process?

### ISSUE THREAD

**Issue by themaster567** [LukeSmithxyz/emailwiz#165]
I just want to completely remove everything this installed and start from scratch. Is that possible or is it too messy? I *should* have kept backups for this sort of thing, but I did not. Is there any way to salvage this situation?

**Comment by LukeSmithxyz** [comment]
`apt purge postfix dovecot-imapd dovecot-sieve opendkim spamassassin spamc opendkim-tools`

Then you can delete remaining directories as well. I.e. `rf -rf /etc/dovecot /etc/postfix /etc/opendkim`

**Comment by themaster567** [comment]
Thank you again. I closed this because I already had a ticket open for something else and didn't want to look too desperate, so especially thank you for responding anyway.

**Comment by ChillerDragon** [comment]
> Then you can delete remaining directories as well. I.e. `rf -rf /etc/dovecot /etc/postfix /etc/opendkim`

lmao i blindly copied your "rf" typo haha

**Comment by themaster567** [comment]
> > Then you can delete remaining directories as well. I.e. `rf -rf /etc/dovecot /etc/postfix /etc/opendkim`
> 
> lmao i blindly copied your "rf" typo haha

Yeah I spotted that and thankfully corrected it lol

**Comment by MoritzMoreira** [comment]
No luck for me, using Raspian OS lite. When I run emailwiz, the option where you enter the domain name does not appear, and at the end, the old domain name is displayed... Will I have to reset my Raspberry?

**Comment by ChillerDragon** [comment]
> No luck for me, using Raspian OS lite. When I run emailwiz, the option where you enter the domain name does not appear, and at the end, the old domain name is displayed... Will I have to reset my Raspberry?

Try editing the `/etc/mailname` file to update your domain

**Comment by MoritzMoreira** [comment]
That was quick :)  Seems to have worked, this time I got the option windows, with the new domain already entered. Just in case: I had also run `purge dovecot,` . The directories did not exist , as before. Unfortunately, I did not achieve to connect Thunderbird until now (no connection to outgoing server). When I try to add a new account, it says "incoming server aleady exists". Restarting the Pi did not help either...
